apps/docs/static/elastic-explorer/elastic-instances.json

```diff
@@ -0,0 +1 @@
+[{"url":"http://ch-casemates-dev-elastic.cz-aws.net:9200","label":"ch casemates dev"},{"url":"http://ch-casemates-dev-v11-elastic.cz-aws.net:9200","label":"ch casemates dev v11"},{"url":"http://ch-legiconso-dev-elastic.cz-aws.net:9200","label":"ch legiconso dev"},{"url":"http://ch-legiconso-dev-v11-elastic.cz-aws.net:9200","label":"ch legiconso dev v11"},{"url":"http://ch-legipro-dev-elastic.cz-aws.net:9200","label":"ch legipro dev"},{"url":"http://ch-mopo-dev-elastic.cz-aws.net:9200","label":"ch mopo dev"},{"url":"http://ch-traites-dev-elastic.cz-aws.net:9200","label":"ch traites dev"},{"url":"http://ep-epdc-dev-elastic.cz-aws.net:9200","label":"ep epdc dev"},{"url":"http://lux-casemates-dev-elastic.cz-aws.net:9200","label":"lux casemates dev"},{"url":"http://lux-legipro-dev-elastic.cz-aws.net:9200","label":"lux legipro dev"},{"url":"http://lux-traites-dev-elastic.cz-aws.net:9200","label":"lux traites dev"}]
```

Comment:
This should contain just one element targeting localhost:9200. We don't want to expose our internal domains in this public repository 😅

Reply:
@WilliamChelman  shouldn't we remove [this](https://github.com/cognizone/ng-cognizone/blob/1ae97ef345e4391ae26a14d7293d0e40a4e6e7ba/apps/docs/static/elastic-explorer/img/jsonUpload.png) screenshot then

Reply:
I will take a new screenshot with 1 entry. (Thanks for pointing out ;))